contributor-guide/modules/ROOT/pages/modular-boost/tortoisegit-library-maintenance.adoc

```diff
@@ -0,0 +1,27 @@
+= TortoiseGit Library Maintenance
```

Comment:
We could confirm that but I don't think separate TortoiseGit is relevant at all anymore. Times have changed and people will either use the command line or whatever comes with their IDE.

Reply:
OK, removed this section.

Reply:
Section is still here. And also the images are still in the commit.